class-tgm-plugin-activation.php

```diff
@@ -3108,6 +3108,7 @@ public function run( $options ) {
 					 *
 					 * @see Plugin_Upgrader::bulk_upgrade()
 					 * @see https://core.trac.wordpress.org/browser/tags/4.2.1/src/wp-admin/includes/class-wp-upgrader.php#L838
+					 * Last synced: Dec 31st 2015 against https://core.trac.wordpress.org/browser/trunk?rev=36134
```

Comment:
For an internal note, it should use the inline format. See https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#78-internal